README.md

```diff
@@ -2,6 +2,13 @@
 These tools allow for annotation of genomic variants from a MAF for import into
 the cBioPortal using [Genome Nexus](http://genomenexus.org)
 
+## Updating the Genome Nexus Annotation Pipeline
```

Comment:
I think this section should move down to the bottom of this document. The other sections tell "What tools" are referred to by the opening section above, so I would not split that.
To emphasize that there is guidance on how to make updates, a table of contents could be added up top with links to the various sections.